### PR TITLE
default settings cleanup

### PR DESCRIPTION
**Pull request by JarbasAl**
optimize config file and requirements.txt

- takes advantage of https://github.com/OpenVoiceOS/ovos-plugin-manager/pull/99 , the best ww plugin will be used depending on what is installed
- no fallback stt by default, this was causing issues in rpi3 because of memory usage
- change default TTS to mimic3 server, mimic2 will go down any day now + voice matches mimic1
- remove ovos-tts-plugin-mimic2 dependency
- remove ovos-tts-plugin-google-tx dependency
- remove ovos-stt-plugin-selene dependency
- remove ovos-stt-plugin-vosk dependency

NOTES:
- precise-lite can still not be made a default because buildtests fail on python 3.10
- stt/tts extra requirements include more plugins not in requirements.txt

TODO:
- adopt https://github.com/OpenVoiceOS/ovos-ww-plugin-openWakeWord as default ww enfine
- remove ovos-ww-plugin-precise dependency
- stt unittests depend on selene plugin, those should be moved to the respective plugin repository, they can also remain here as integration tests

### CODE REVIEW

File: requirements/extra-stt.txt [hunk 1]
```diff
@@ -1,7 +1,12 @@
 SpeechRecognition~=3.8
 PyAudio~=0.2
+
 ovos-vad-plugin-webrtcvad~=0.0.1
+
 ovos-ww-plugin-pocketsphinx~=0.1, >=0.1.3
 ovos-ww-plugin-precise~=0.1
-ovos-stt-plugin-selene>=0.0.3a3
+ovos-ww-plugin-precise-lite
```
Comment (NeonDaniel) [inline requirements/extra-stt.txt:8]: consider adding version specs here

File: requirements/extra-tts.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
 ovos-tts-plugin-mimic~=0.2, >=0.2.6
 ovos-tts-plugin-mimic2~=0.1, >=0.1.5
-ovos-tts-plugin-google-tx~=0.0, >=0.0.3
\ No newline at end of file
+ovos-tts-plugin-mimic3-server
```
Comment (NeonDaniel) [inline requirements/extra-tts.txt:3]: Consider version spec

File: requirements/requirements.txt [hunk 3]
```diff
@@ -18,15 +18,15 @@ padaos~=0.1
...
+ovos-plugin-manager~=0.0, >=0.0.22
 ovos_workshop~=0.0, >=0.0.11
 ovos_PHAL~=0.0, >=0.0.2
 ovos-lingua-franca>=0.4.6
 
 ovos-stt-plugin-server~=0.0, >=0.0.2
 ovos-tts-plugin-mimic~=0.2, >=0.2.6
-ovos-tts-plugin-mimic2~=0.1, >=0.1.5
-ovos-tts-plugin-google-tx~=0.0, >=0.0.3
+ovos-tts-plugin-mimic3-server
```
Comment (NeonDaniel) [inline requirements/requirements.txt:28]: Consider version spec

File: requirements/requirements.txt [hunk 4]
```diff
@@ -41,3 +41,9 @@ ovos-ocp-news-plugin~=0.0.3
 ovos-skill-volume~=0.0.1
 ovos-skill-fallback-unknown~=0.0.2
 ovos-skill-stop~=0.2
+neon-skill-alerts~=1.2
+ovos-skill-personal
+ovos-skill-naptime
+ovos-skill-date-time>=0.2.2a1
+ovos-skill-hello-world
+ovos-skill-filebrowser
```
Comment (NeonDaniel) [inline requirements/requirements.txt:49]: Could go in a separate PR, but these should have stable release versions

File: requirements/tests.txt [hunk 5]
```diff
@@ -6,5 +6,7 @@ cov-core==1.15.0
 sphinx==2.2.1
 sphinx-rtd-theme==0.4.3
 mock_msm~=0.9.2
+
+ovos-stt-plugin-selene>=0.0.3a3
```
Comment (NeonDaniel) [inline requirements/tests.txt:10]: Same as above, this should have a stable version spec but could be handled in a separate PR

File: mycroft/configuration/mycroft.conf [hunk 6]
```diff
@@ -540,10 +562,9 @@
   // Speech to Text parameters
   // Override: REMOTE
   "stt": {
-    // Engine.  Options: "mycroft", "google", "wit", "ibm", "kaldi", "bing",
-    //                   "houndify", "deepspeech_server", "govivace", "yandex"
+    // select a STT plugin as described in the respective readme
     "module": "ovos-stt-plugin-server",
-    "fallback_module": "ovos-stt-plugin-vosk",
```
[the remark this answers is not in the document]
Comment (JarbasAl) [reply mycroft/configuration/mycroft.conf:546]: platform support concerns for all offline options, rpi3 support in particular proved cumbersome for end users because this default value would eat up all memory

**Comment by j1nx** [comment]
Apologies for not reviewing in time
